Sources/SWBUtil/FSProxy.swift

```diff
@@ -562,9 +562,13 @@ class LocalFS: FSProxy, @unchecked Sendable {
     }
 
     func remove(_ path: Path) throws {
+        #if os(Windows)
```

Comment:
I don't know that there's any value in this being platform-conditional. I think this is only using unlink in the first place out of some early attempt to avoid Foundation for whatever reason at the time which is long since irrelevant. And we're using FileManager in the implementation of most of these FSProxy these functions anyways, so we may as well get to benefit from Foundation's work.